src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -28,6 +28,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val help = BooleanSetting("-help", "Print a synopsis of standard options")
   val nowarn = BooleanSetting("-nowarn", "Generate no warnings.")
   val print = BooleanSetting("-print", "Print program with Scala-specific features removed.")
+  val color = ChoiceSetting("-color", "mode", "Colored output", List("always", "never", "auto"), "auto")
```

Comment:
@smarter: done. For now it defaults to "auto" and the repl will be colored if the setting is set to "auto" or "always".